core/src/main/java/org/jobrunr/dashboard/JobRunrApiHandler.java

```diff
@@ -109,7 +109,10 @@ private HttpRequestHandler getRecurringJobs() {
 
     private HttpRequestHandler deleteRecurringJob() {
         return (request, response) -> {
-            storageProvider.deleteRecurringJob(request.param(":id"));
+            int deleted = storageProvider.deleteRecurringJob(request.param(":id"));
+            if(deleted == 0) {
+                response.statusCode(404);
```

Comment:
Should this a throw a `JobNotFoundException` and we let the exception handler what to do with it?